extensions/objects/templates/FilepathEditor.js

```diff
@@ -36,6 +36,11 @@ oppia.directive('filepathEditor', [
         $scope.MODE_SAVED = 3;
         $scope.MAX_OUTPUT_IMAGE_WIDTH_PX = 490;
 
+        // We only use PNG format since that is what canvas can export to in
+        // all browsers.
+        // TODO(sll): See if we can add support for other image formats.
+        var RESAMPLED_IMAGE_FORMAT = 'png';
```

Comment:
I declared all the constants as:
$scope.MY_CONSTANT = ...;

Perhaps I shouldn't have. I should have done what you did (var MY_CONSTANT), but since I did, I would prefer that we follow the same pattern for consistency.

Wdyt?